src/Controls/src/Core/Platform/AlertManager/DialogHelper.Gtk.cs

```diff
@@ -0,0 +1,150 @@
+using System;
+using System.Linq;
+using GLib;
+using Gtk;
+using Microsoft.Maui.Controls.Internals;
+
+
+namespace Microsoft.Maui.Controls.Platform
+{
+	// Ported from https://github.com/nblockchain/DotNetForms/blob/5.0.0/Xamarin.Forms.Platform.GTK/Helpers/DialogHelper.cs
```

Comment:
@webwarrior-ws let's use a URL from github.com/xamarin/Xamarin.Forms/, not DotNetForms